.ci/scripts/send-pr.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+readonly APM_AGENT=${1}
+readonly APM_AGENT_SPECS_DIR=${2}
+readonly APM_AGENT_REPO_NAME="apm-agent-${APM_AGENT}"
+readonly GIT_DIR=".ci/git"
+readonly APM_AGENT_REPO_DIR="${GIT_DIR}/${APM_AGENT_REPO_NAME}"
+
+git clone "https://github.com/elastic/${APM_AGENT_REPO_NAME}" "${APM_AGENT_REPO_DIR}"
+
+mkdir -p "${APM_AGENT_REPO_DIR}/${APM_AGENT_SPECS_DIR}"
+cp tests/agents/gherkin-specs/*.feature "${APM_AGENT_REPO_DIR}/${APM_AGENT_SPECS_DIR}"
+
+cd ${APM_AGENT_REPO_DIR}
+git checkout -b update-feature-files-$(date "+%Y%m%d%H%M%S")
+echo "Copying feature files to the ${APM_AGENT_REPO_NAME} repo"
+git status
+git add ${APM_AGENT_SPECS_DIR}
```

Comment:
Ideally, the git user.name and user.email should be in place (see PR 17214 in the infra repo), if that's not the case, then you might get some warnings, or maybe some errors, although I don't recall now.

I'd say to use some validation in case they are not defined to set them. what do you think?

Reply:
Instead of a validation, I have added two `gitCmd` calls to ensure they are present in the build agent (see 08f9d9ac491ca046c6681b8fb085769852329416). wdyt?

Reply:
I started with the check, but I realised that it is worth it to be prescriptive in this case, where we do not want other configuration. Just in case, that was my initial validation, at script level:

```shell
readonly GIT_USERNAME="elasticmachine"
readonly GIT_EMAIL="infra-root-${GIT_USERNAME}@elastic.co"

function checkGitConfig {
    local config="${1}"
    local defaultValue="${2}"

    git config "${config}"
    local exitCode=$?
    if [ ${exitCode} -ne 0 ]; then
        git config "${config}" "${defaultValue}"
    fi
}

function checkGit {
    checkGitConfig "user.name" "${GIT_USERNAME}"
    checkGitConfig "user.email" "${GIT_EMAIL}"
}
```